content/millennium/r4/clinical/care-provision/nutrition-order.md

```diff
@@ -16,6 +16,7 @@ The following fields are returned if valued:
 * [NutritionOrder id](https://hl7.org/fhir/r4/resource-definitions.html#Resource.id)
 * [Status](https://www.hl7.org/fhir/r4/nutritionorder-definitions.html#NutritionOrder.status)
 * [Intent](https://www.hl7.org/fhir/r4/nutritionorder-definitions.html#NutritionOrder.intent)
+* [Extensions including basedOn](#extensions){:target="_blank"}
```

Comment:
You need the target here because it's an internal link? The others are missing it

Reply:
[commit](https://github.com/cerner/fhir.cerner.com/pull/679/commits/9a3f36a3301560f863aaecdc43469ac05c6d2a46#diff-63c407fea7510167107b85f2810bc5775ff46200c2554d092fcb3df7a3ea2aaeL19)